default.conf

```diff
@@ -0,0 +1,5 @@
+location / {
```

Comment:
I think you need to include the entire NGINX default config and change the one line, right? This will just have the 5 line file and Nginx will refuse to launch I'm fairly sure.

Thanks!

Reply:
Do you have the config at hand?

Reply:
[nginx.zip](https://github.com/PhlexPlexico/G5V/files/7017117/nginx.zip)

Here, may have to `dos2unix` before uploading it, and the path is `/etc/nginx/nginx.conf`, not `/etc/nginx/conf.d/default.conf`.

After that it should be good!

Reply:
```nginx
user  nginx;
worker_processes  auto;

error_log  /var/log/nginx/error.log notice;
pid        /var/run/nginx.pid;


events {
    worker_connections  1024;
}


http {
    include       /etc/nginx/mime.types;
    default_type  application/octet-stream;

    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';

    access_log  /var/log/nginx/access.log  main;

    sendfile        on;
    #tcp_nopush     on;

    keepalive_timeout  65;

    #gzip  on;

    include /etc/nginx/conf.d/*.conf;
}
```
_just so we have it here_

Reply:
Following [this](https://adamtheautomator.com/nginx-docker/) guide, it seems to be enough to just have a `server` block

Reply:
While true, this is direct from a docker instance that is currently live in the wild. I would just add in the `try_files` right before the `include` and call it a day, we only wish to append, not to completely modify :)

Reply:
sou you would propose smth like this?
```nginx
user  nginx;
worker_processes  auto;

error_log  /var/log/nginx/error.log notice;
pid        /var/run/nginx.pid;


events {
    worker_connections  1024;
}

http {
    include       /etc/nginx/mime.types;
    default_type  application/octet-stream;

    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';

    access_log  /var/log/nginx/access.log  main;

    sendfile        on;
    #tcp_nopush     on;

    keepalive_timeout  65;

    #gzip  on;
    
    try_files $uri /index.html;

    include /etc/nginx/conf.d/*.conf;
}
```

Reply:
What you have there now should work perfectly I believe!